OMCompiler/Compiler/Util/Flags.mo

```diff
@@ -160,6 +160,13 @@ public constant Integer METAMODELICA = 2;
 public constant Integer PARMODELICA = 3;
 public constant Integer OPTIMICA = 4;
 public constant Integer PDEMODELICA = 5;
+
+// FMI-ModelDescription-ENUM-FLAGS
+public constant Integer FMI_NONE = 6;
```

Comment:
This should start at 1, even though it actually doesn't matter at all.

Reply:
updated the values to start from 1